### ISSUE TITLE
`rename()` should allow lists of names to be changed (in addition to individual values)

### ISSUE THREAD

**Issue by rbyh** [expectedparrot/edsl#702]

**Comment by johnjosephhorton** [comment]
I actually did this w/ a rename_dict (?) that lets you pass a dictionary
but I'm not in love w/ that and I'd probably prefer having one general
purpose rename method...

**Comment by rbyh** [comment]
Yeah I saw dict but then you need a step changing your list to dict which is not ideal
